hrui/css/hrui.css

```diff
@@ -2,50 +2,8 @@
  * Ensures that there is the same top padding regardless whether
  * the user can see the admin toolbar or not
  */
-@font-face {
-  font-family: 'icomoon';
-  src:url('../fonts/icomoon.eot?iqn065');
-  src:url('../fonts/icomoon.eot?#iefixiqn065') format('embedded-opentype'),
-  url('../fonts/icomoon.woff?iqn065') format('woff'),
-  url('../fonts/icomoon.ttf?iqn065') format('truetype'),
-  url('../fonts/icomoon.svg?iqn065#icomoon') format('svg');
-  font-weight: normal;
-  font-style: normal;
-}
-
-
-@media screen and (-webkit-min-device-pixel-ratio: 0) {
-  @font-face {
-    font-family: 'icomoon';
-    src: url('../fonts/icomoon.svg?iqn065#icomoon') format('svg');
-  }
-}
-
-[class^="icon-"], [class*=" icon-"] {
-  font-family: 'icomoon';
-  speak: none;
-  font-style: normal;
-  font-weight: normal;
-  font-variant: normal;
-  text-transform: none;
-  line-height: 1;
-
-  /* Better Font Rendering =========== */
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-.icon-logo-full:before {
-  content: "\e601";
-}
-
-.icon-logo-full:before {
-  color: #727f8a;
-  font-size: 22px;
-}
-
-body:not(.toolbar-drawer) {
-  padding-top: 30px;
+body {
+  padding-top: 60px !important;
```

Comment:
Why `!important` ? I see a commit [PCHR-2609: Enforce new padding to body in all circumstances](https://github.com/civicrm/civihr/pull/2184/commits/1aa20f0df979ee240a29d551de161b26cc4e7434), but what are the circumstances that would overwrite the padding-top of the page? Can't they be fixed?

Reply:
Drupal applies a top padding to the body via javascript, that's why we have to override it via css with an `!important`